src/main/java/bral/dictionary/DictionaryFrame.java

```diff
@@ -0,0 +1,78 @@
+package bral.dictionary;
+
+import com.opencsv.exceptions.CsvValidationException;
+import javax.swing.*;
+import java.awt.*;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+import java.io.IOException;
+import java.util.List;
+
+public class DictionaryFrame extends JFrame
+{
+
+    private final EnglishDictionary englishDictionary = new EnglishDictionary();
+
+
+    private JTextField searchField;
+
+    private JTextArea definitionsTextArea;
+
+    public DictionaryFrame() throws CsvValidationException, IOException
+    {
+        setSize(1000, 800);
+        setTitle("English Dictionary");
+
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        JPanel main = new JPanel();
+        main.setLayout(new BorderLayout());
+        // tells the JFrame to use this JPanel
+        setContentPane(main);
+
+
+        searchField = new JTextField("");
+        main.add(searchField, BorderLayout.NORTH);
+
+        definitionsTextArea = new JTextArea("Definitions will show here...");
+        definitionsTextArea.setEditable(false);
+        definitionsTextArea.setLineWrap(true);
+        definitionsTextArea.setWrapStyleWord(true);
+        main.add(definitionsTextArea, BorderLayout.CENTER);
+
+        JScrollPane scrollPane = new JScrollPane(definitionsTextArea);
+        main.add(scrollPane);
```

Comment:
You didn't answer my question and instead changed a different line that I wasn't even talking about.

Reply:
Thank you for the feedback. So I should remove the center for the definitionsTextArea? I'm not really sure what the problem is since it looks the same with or without it

Reply:
I didn't ask you to remove anything. 

I simply asked why you are adding it to the center. 

You just ignored the question. 

I'm asking you to explain the code you've written.

Reply:
Only one thing can be in the center. You put 2 things in the center. The second one replaces the first.

Reply:
Okay got it, thank you